### ISSUE TITLE
Would like to get JWT token wihout having to stop Hornet

### ISSUE THREAD

**Issue by nuriel77** [iotaledger/hornet#1123]
**Is your feature request related to a problem? Please describe.**
It seems that the `tool jwt` command needs a lock on the p2p DB, that means that Hornet has to be stopped in order to get the token.

Is it possible for it not to request the lock on the DB, as this a READ operation only, no?

**Describe the solution you'd like**
Be able to get JWT via tool jwt without having to stop Hornet.

**Environment**
I've only tested this on Docker so far, but I guess the same is true for /bin/hornet on the OS.

**Current Output**
```
panic: unable to initialize data store for peer store: Cannot acquire directory lock on "/app/p2p/store".  Another process is using this Badger database.: resource temporarily unavailable
```

**Comment by muXxer** [comment]
I spent quite some time (2 days) some weeks ago to achieve that functionality.
Unfortunately BadgerDB is such a scam :(
Even with ReadOnly access and OverwriteLocking, we can't open the database in parallel, and so we can't create new tokens.

Is this really that important? I mean, this is kind of a one time thing, no?

**Comment by nuriel77** [comment]
> Is this really that important? I mean, this is kind of a one time thing, no?

I guess it is a Nice-to-have. It isn't a blocker nor a bug.
Maybe just a note in the documentation would be good for now, I was approach by some users who didn't understand why they couldn't get the token.

Thanks.

**Comment by muXxer** [comment]
Fixed with #1167
